src/tasks/onetime/DeliveryTask.py

```
@@ -806,3 +806,59 @@ def run(self):
+        self.try_time = 0
+        self._configure_delivery_area(DEFAULT_DELIVERY_AREA)
+
+        task.default_config.update({
+            self.DAILY_ENABLE_KEY: True,
+            self.CFG_TARGET_TICKET_NUM: ["119000"],
+            self.CFG_DELIVERY_AREA: DEFAULT_DELIVERY_AREA,
+        })
+        task.config_description.update({
+            self.DAILY_ENABLE_KEY: "是否执行自动接取并完成运输委托。",
+            self.CFG_TARGET_TICKET_NUM: (
+                "目标券数优先级序列，用逗号分隔多个券数。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)